numpy/random/tests/test_random.py

```diff
@@ -440,6 +440,14 @@ def test_choice_return_shape(self):
         assert_equal(np.random.choice(6, s, replace=False, p=p).shape, s)
         assert_equal(np.random.choice(np.arange(6), s, replace=True).shape, s)
 
+        # Check zero-size
+        assert_equal(np.random.randint(0, 0, size=(3, 0, 4)).shape, (3, 0, 4))
```

Comment:
A test for `randint(10, 10, size=0)` might be good too - just to check that we didn't special case 0 somehow

Reply:
done